api-server/common/models/user.js

```diff
@@ -565,14 +571,15 @@ export default function(User) {
   };
 
   User.prototype.updateMyUsername = function updateMyUsername(newUsername) {
+    console.log(newUsername);
```

Comment:
```suggestion
    
```
Just a reminder comment to remove this later, when you are ready with the PR.

Reply:
@raisedadead removed console.log